po/fr.po

```diff
@@ -25129,1346 +25075,1362 @@ msgstr "Exécution en silence"
+msgstr "Lire les validations depuis l’entrée standard"
 
 #: /tmp/fish/implicit/share/completions/git.fish:177
 msgid "Mark equivalent commits with = and inequivalent with +"
-msgstr ""
+msgstr "Marquer les validations équivalentes avec = et non équivalentes avec +"
 
 #: /tmp/fish/implicit/share/completions/git.fish:178
 msgid "Omit equivalent commits"
-msgstr ""
+msgstr "Ignorer les validations équivalents"
 
 #: /tmp/fish/implicit/share/completions/git.fish:179
 msgid "Join two or more development histories together"
-msgstr ""
+msgstr "Fusionner plusieurs historiques de développement"
 
 #: /tmp/fish/implicit/share/completions/git.fish:182
 msgid "Autocommit the merge"
-msgstr ""
+msgstr "Auto-valider la fusion"
 
 #: /tmp/fish/implicit/share/completions/git.fish:183
 msgid "Don't autocommit the merge"
-msgstr ""
+msgstr "Ne pas auto-valider la fusion"
 
 #: /tmp/fish/implicit/share/completions/git.fish:184
 msgid "Edit auto-generated merge message"
-msgstr ""
+msgstr "Éditer le message de fusion auto-généré"
 
 #: /tmp/fish/implicit/share/completions/git.fish:185
 msgid "Don't edit auto-generated merge message"
-msgstr ""
+msgstr "Ne pas éditer le message de fusion auto-généré"
 
 #: /tmp/fish/implicit/share/completions/git.fish:186
 msgid "Don't generate a merge commit if merge is fast-forward"
 msgstr ""
+"Ne pas générer de validation de fusion si la fusion est faite en avance "
+"rapide"
 
 #: /tmp/fish/implicit/share/completions/git.fish:187
 msgid "Generate a merge commit even if merge is fast-forward"
 msgstr ""
+"Générer un validation de fusion même si la fusion est faite en avance rapide"
 
 #: /tmp/fish/implicit/share/completions/git.fish:188
 msgid "Refuse to merge unless fast-forward possible"
-msgstr ""
+msgstr "Refuser la fusion sauf si une avance rapide est possible"
 
 #: /tmp/fish/implicit/share/completions/git.fish:189
 msgid "Populate the log message with one-line descriptions"
-msgstr ""
+msgstr "Remplir le message de journal avec des descriptions sur une ligne"
 
 #: /tmp/fish/implicit/share/completions/git.fish:190
 msgid "Don't populate the log message with one-line descriptions"
 msgstr ""
+"Ne pas remplir le message de journal avec des descriptions sur une ligne"
 
 #: /tmp/fish/implicit/share/completions/git.fish:191
-#, fuzzy
 msgid "Show diffstat of the merge"
-msgstr "Afficher l'état des dépendances"
+msgstr "Montrer le diffstat de la fusion"
 
 #: /tmp/fish/implicit/share/completions/git.fish:192
 msgid "Don't show diffstat of the merge"
-msgstr ""
+msgstr "Ne pas montrer le diffstat de la fusion"
 
 #: /tmp/fish/implicit/share/completions/git.fish:193
 msgid "Squash changes from other branch as a single commit"
 msgstr ""
+"Compresser les modifications depuis une autre branche comme une seule "
+"validation"
 
 #: /tmp/fish/implicit/share/completions/git.fish:194
-#, fuzzy
 msgid "Don't squash changes"
-msgstr "Pas de changements"
+msgstr "Ne pas compresser les changements"
```

Comment:
Inconsistent terminology for changes: ”modifications” (above) versus “changements” (this message).